tsconfig.json

```diff
@@ -36,10 +36,17 @@
     },
     "incremental": true
   },
-  "exclude": ["node_modules"],
+  "exclude": [
+    "node_modules",
+    "*/**/*.cy.ts" /* .cy.test files are covered by cypress/tsconfig */
+  ],
   "include": [
     "./src",
     "src/api",
     "src/shared",
-    "next-env.d.ts", "**/*.ts", "**/*.tsx", "next.config.js"],
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
```

Comment:
given these two lines, do you still need lin 44, 45, 46, 47, 50? can you remove them?

also, if to keep line 44, change it to `src`.